components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -287,7 +296,7 @@ export default class GalleryItem extends Vue {
           ...nFTEntity,
           emotes: nFTEntity?.emotes?.nodes,
         }),
-        result: () => this.fetchMetadata(),
+        result: () => { this.fetchMetadata(); this.fetchCollectionItems() },
```

Comment:
as these 2 functions are async functions, you can consider using Promise.all() or Promise.allSettled()